### ISSUE TITLE
Update Jetbrains marketplace with new release of plug-in

### ISSUE THREAD

**Issue by pdaverh** [redhat-developer/intellij-dependency-analytics#1]
The overview content shown in Jetbrains marketplace for the IDE plug-in (https://plugins.jetbrains.com/plugin/12541-dependency-analytics) should be similar to that of VS Code, except the stack analysis report capability which is not available. The overview should include screenshots/gifs of the analysis offered by the plug-in in JetBrains IDEs. The version number of the release should match with VS Code Marketplace.

**Comment by jeffmaury** [comment]
Agree to the screenshots but would the version match as the features are not aligned ?

**Comment by pdaverh** [comment]
True. Since the release announcement has a version number, thought it would be better to keep the versions number in sync across all IDEs.
